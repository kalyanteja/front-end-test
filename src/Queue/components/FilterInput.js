import styled from 'styled-components';

export default styled.input`
  padding: 0.5em;
  margin: 0.5em;
  background: lightgray;
  border: none;
  border-radius: 3px;

  @media (max-width: 450px){
      width: 90%;
  }
  
  @media (min-width: 450px){
    margin-left: 10%;
    width: 75%;
  }
`;