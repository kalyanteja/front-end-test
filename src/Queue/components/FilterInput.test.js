import React from 'react';
import FilterInput from './FilterInput';
import renderer from 'react-test-renderer';

describe('filter input component testing', () => {
  it('renders filter input styled-component', () => {
    const tree = renderer
    .create(<FilterInput></FilterInput>)
    .toJSON();
    expect(tree).toMatchSnapshot()
  });
})

