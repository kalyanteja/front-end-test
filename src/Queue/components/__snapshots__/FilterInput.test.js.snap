// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filter input component testing renders filter input styled-component 1`] = `
<input
  className="sc-bdVaJa cZawRP"
/>
`;
